tools/prkit/verify.sh

```
@@ -0,0 +1,153 @@
+
+# --- 8. Root scaffold files (outside the SCAN roots) — present, non-empty, valid ---
+for req in README.md LICENSE NOTICE CHANGELOG.md .gitignore \
+           .github/workflows/ci.yml .claude-plugin/marketplace.json; do
+  [ -s "$ROOT/$req" ] || fail "scaffold: missing/empty $req"
+done
+jq empty "$ROOT/.claude-plugin/marketplace.json" 2>/dev/null || fail "scaffold: marketplace.json is not valid JSON"
+ok "scaffold: root files present, non-empty, marketplace.json valid"
+
+# --- 9. No unrendered template placeholders leaked into generated output ---
+PLH=$(grep -rlE '\{\{[A-Za-z_]+\}\}' "${SCAN[@]}" "$ROOT/README.md" "$ROOT/CHANGELOG.md" "$ROOT/.claude-plugin/marketplace.json" 2>/dev/null || true)
+[ -z "$PLH" ] && ok "placeholders: no unrendered {{...}} in output" || { fail "placeholders: unrendered {{...}} in:"; echo "$PLH" | sed 's/^/         /'; }
```

Shellcheck (0.11.0)

[info] Note that A && B || C is not if-then-else. C may run when A is true.

(SC2015)